core/src/it/scala/io/km8/core/kafka/KafkaExplorerSpec.scala

```diff
@@ -22,7 +21,6 @@ object KafkaExplorerSpec extends DefaultRunnableSpec:
 
         val layer =
           Clock.live >+>
-            Blocking.live >+>
             itlayers.kafkaContainer >+>
```

Comment:
Shouldn't we skip over composing layers live this and use the `ZLayer.make` with automatic injection?